src/claim/Claim.js

```diff
@@ -319,12 +338,14 @@ class Claim extends UserCollectableAttribute {
         // Properties is not an object
         properties.push(`${basePropName}.${typeDefProps.name}`);
       } else {
-        _.forEach(typeDefProps, (prop) => {
+        // eslint-disable-next-line no-restricted-syntax
+        for (const prop of typeDefProps) {
           const { isNewIdentifier } = getBaseIdentifiers(prop.type);
           const newBasePropName = !isNewIdentifier ? basePropName : `${basePropName}.${prop.name}`;
-          const proProperties = this.getAllProperties(prop.type, newBasePropName);
+          // eslint-disable-next-line no-await-in-loop
```

Comment:
Why do eslint disable, beats the purpose of even having the lint on the first place